staging_service/import_specifications/individual_parsers.py

```diff
@@ -316,3 +317,50 @@ def parse_excel(path: Path) -> ParseResults:
         return ParseResults(frozendict(results))
     else:
         return _error(Error(ErrorType.PARSE_FAIL, "No non-header data in file", spcsrc))
+
+
+def parse_dts_manifest(path: Path) -> ParseResults:
+    """
+    Parse the provided DTS manifest file. Expected to be JSON, and will fail otherwise.
+    The manifest should have roughly this format, with expected keys included:
+    {
+        "resources": [{
+            "id": str,
+            "name": str,
+            "path": str,
+            "format": str,
+            "instructions": {
+                "data_type": str,
+                "parameters": {
+                    "<param1>": value,
+                    "<param2>": value
+                }
+            }
+        }]
+    }
+    The parameters under the "instructions"."parameters" dictionary are arbitrary keys with
+    arbitrary values and will be returned as-is. They are expected to be PRIMITIVE_TYPEs and
+    will fail otherwise.
+    """
+    spcsrc = SpecificationSource(path)
+    errors = []
+    # dummy for now
+    results = {}
+    try:
+        with open(path, "r") as manifest:
+            manifest_json = json.load(manifest)
+        if not isinstance(manifest_json, dict):
+            errors.append(Error(ErrorType.PARSE_FAIL, "Manifest is not a dictionary", spcsrc))
```

Comment:
You might want to consider jsonschema for checking the structure

Reply:
Next PR